enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/messaging/CoreReplicatedContentMarshal.java

```diff
@@ -85,6 +87,11 @@ else if ( content instanceof DistributedOperation )
             channel.put( DISTRIBUTED_OPERATION );
             ((DistributedOperation) content).serialize( channel );
         }
+        else if ( content instanceof DummyRequest )
```

Comment:
I think that could make a lot of sense! A lot of this code is quite old and could use a refactoring.